### ISSUE TITLE
Kaart edits in Scotland

### ISSUE THREAD

**Issue by chrisfleming** [osm-scotland/scotland-issues#7]
https://osmcha.mapbox.com/changesets/68153090/
https://osmcha.mapbox.com/changesets/68154129/
https://osmcha.mapbox.com/changesets/68075919/
https://osmcha.mapbox.com/changesets/68175673/
https://www.openstreetmap.org/changeset/68264555
https://osmcha.mapbox.com/changesets/68266425/

**Comment by boothym** [comment]
Are they just deleting the highway areas that someone has added a lot of in the Borders?

**Comment by chrisfleming** [comment]
Yes - some of the changes are okay, and some of the areas are perhaps excessive; but some of the area's should be kept. I also noticed them removing other details like fences on some of the changes.

**Comment by chrisfleming** [comment]
Also https://osmcha.mapbox.com/changesets/68271595/
https://osmcha.mapbox.com/changesets/68272220/
https://osmcha.mapbox.com/changesets/68272479/
https://osmcha.mapbox.com/changesets/68074331/

**Comment by chrisfleming** [comment]
Have had a reply to Changeset comment: https://www.openstreetmap.org/changeset/68264555#map=15/55.7227/-2.7368

Also comments on https://www.openstreetmap.org/changeset/68154129

**Comment by chrisfleming** [comment]
https://overpass-api.de/achavi/?changeset=68378338
